tests/integration/test_interchain.py

```diff
@@ -174,6 +190,42 @@ def test_create_bitcoin_interchain_fails_with_bad_private_key(self):
             response,
         )
 
+    def test_create_binance_interchain_with_values(self):
+        url = "http://binance-node.dragonchain.com"
+        privkey = "c7603e7928cc9892982d5cda97cc50b9db81ec40ef3f7b9d515fb333151140bf"
+        response = self.client.create_binance_interchain(
+            name=BINANCE_INTERCHAIN_NAME_1, testnet=True, private_key=privkey, node_url=url, rpc_port=26657, api_port=11699
+        )
+        self.assertEqual(response, {"status": 201, "ok": True, "response": BINANCE_INTERCHAIN_BODY}, response)
+
+    def test_create_binance_interchain_with_defaults(self):
+        response = self.client.create_binance_interchain(name=BINANCE_INTERCHAIN_NAME_2)
+        self.assertTrue(response.get("ok"), response)
+        self.assertEqual(response.get("status"), 201, response)
+        jsonschema.validate(response.get("response"), schema.binance_interchain_at_rest_schema)
+
+    def test_create_binance_interchain_fails_with_bad_node_address(self):
+        response = self.client.create_binance_interchain(
+            name="garbage", testnet=True, node_url="http://youcantGETthis.whatever", rpc_port=26657, api_port=11699
+        )
+        self.assertFalse(response.get("ok"), response)
+        self.assertEqual(response.get("status"), 400, response)
+        self.assertTrue(response.get("response")["error"]["details"].startswith("Provided binance node doesn't seem reachable."), response)
+
+    def test_create_binance_interchain_fails_with_bad_private_key(self):
+        bad_hex = "9iQWROmdJ3iGwIZVDghnp2WpIejd0dpb9KPvgDEA06N3B0zp9CGOW5yAf09f8d8c"  # 64 random chars
+        response = self.client.create_binance_interchain(name="garbage", testnet=True, private_key=bad_hex)
+        BINANCE_INTERCHAIN_TEST(bytearray.fromhex(bad_hex[-8:]).decode())
```

Comment:
?

Reply:
Once this line is deleted along with the assignment of print, it all looks good

Reply:
forgot that, thanks -- removed